frontend-site/src/modules/LugApi.js

```diff
@@ -38,4 +50,5 @@ export class LugApi {
   }
 }
 
-export default new LugApi(window.webpackHotUpdate !== undefined ? 'http://localhost:5000' : '');
+// use localhost for dev purposes, current domain for production
+export default new LugApi(isDebugMode() ? 'http://localhost:5000' : '');
```

Comment:
At some point we should make the development port configurable. Probably not in this PR though.